pkg/message/nats/nats.go

```diff
@@ -0,0 +1,44 @@
+package nats
```

Comment:
Ideally this needs to be marshalled to an interface. Can you look at `meshkit/config` for reference? I'll merge it for now